src/commands/farm.rs

```diff
@@ -153,45 +152,17 @@ async fn start_farming(farming_args: FarmingArgs) -> Result<(Farmer, Node)> {
 async fn prepare_farming() -> Result<FarmingArgs> {
     let config = validate_config()?;
 
-    let node_config = config.node;
-    let NodeConfig {
-        chain,
-        execution: _,
-        blocks_pruning,
-        state_pruning,
-        role,
-        name,
-        listen_addresses,
-        rpc_method,
-        force_authoring,
-    } = node_config;
-
-    let chain = match chain.as_str() {
+    let chain = match config.chain.as_str() {
```

Comment:
A side note. You can add type for that which would implement `Serialize, Deserialize`. Smth like
```rust
pub enum ChainConfig {
    #[default]
    Gemini3A,
    Dev,
}
```